src/coreclr/vm/codeversion.h

```diff
@@ -74,12 +74,14 @@ class NativeCodeVersion
     enum OptimizationTier
     {
         OptimizationTier0,
+        OptimizationTier0Instrumented,
```

Comment:
As this enum is converted to an integer and directly emitted to the eventtrace stream as part of the jit information, we cannot change the values of existing enum entries such as OptimizationTier1. You need to put this enum value at the end of the OptimizationTier enum. Also, please add a comment about this to the source code.

Reply:
Done